src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+[class*="block"] {
```

Comment:
better to create class instead of this structure `[class*="block"]`.
It would be easier to read and debug it